src/auction.js

```diff
@@ -582,7 +582,9 @@ function tryAddVideoBid(auctionInstance, bidResponse, afterBidAdded, {index = au
     }), 'video');
   const context = videoMediaType && deepAccess(videoMediaType, 'context');
   const useCacheKey = videoMediaType && deepAccess(videoMediaType, 'useCacheKey');
+  const [hasTrackers, vastTrackers] = getVastTrackers(bidResponse);
 
+  if (hasTrackers) { bidResponse.vastXml = insertVastTrackers(vastTrackers, bidResponse.vastXml); }
```

Comment:
let's make this dependent on configuration as well?

Reply:
something like that to add in config ?
`pbjs.setConfig({ videoTrackers: true });`
If so, shouldn't it be configuration dependant here as well in `videoCache.js` ?
`if (Array.isArray(arrayVastTrackers) && arrayVastTrackers.length == 2 && arrayVastTrackers[1].hasOwnProperty('impressions')) {`
What default value should be set in config.js ?

Reply:
Hi @patmmccann , can you confirm me that I understood what you asked for so I can make the changes ? Thanks

Reply:
Any feedback on this please ?